staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness_test.go

```diff
@@ -510,6 +806,8 @@ func newHandlerChain(t *testing.T, handler http.Handler, filter utilflowcontrol.
 	})
 
 	handler = WithTimeoutForNonLongRunningRequests(handler, longRunningRequestCheck)
+	// we don't have any request with invalid timeout, so leaving audit policy and sink nil.
+	handler = apifilters.WithRequestDeadline(handler, nil, nil, longRunningRequestCheck, nil, requestTimeout)
```

Comment:
are you worried about this falling out of sync with the real one?

Reply:
This is a unit test, so I think it is OK to not match the full real handler chain.